File: Assignment13/Program2.c
//Prbolem statement :Accept N numbers from user and display all such elements which are divisible by 5
/*
Input : 6
       Element : 85 66 3 80 93 88

Ouput : 85 80       
*/
#include<stdio.h> // printf and scanf
#include<stdlib.h>//For malloc and free

void Display(int Arr[] ,int iLength)
{
    register int iCnt = 0; 
    for(iCnt = 0 ; iCnt < iLength ; iCnt++)
    {
        if(Arr[iCnt] % 5 == 0)
        {
            printf("%d\t\n",Arr[iCnt]);
        
        }
    }
    
}


int main()

{

 register int iCnt = 0;
 auto int iSize = 0;
 auto int *p = NULL;  

 
  
 printf("Enter Number of elements :\n");
 scanf("%d",&iSize);

 p = (int *) malloc(iSize * sizeof(int)); //malloc call

 printf("Dynamic memory gets allocated sucessfully for %d Elements\n",iSize); 

 if(p == NULL)
{
    printf("Unable to allocate Memory ");
    return -1;
}

 printf("Enter %d elements " ,iSize);

 for (iCnt =0; iCnt < iSize; iCnt++)
 {
     printf("\nEnter Element %d :",iCnt + 1 );
     scanf("%d", &p[iCnt]);
 }
 Display(p,iSize); //Function Call

 free(p);//to deallocate the memory
 printf("Dynamic memory gets deallocated sucessfully......\n");
    return 0;

}

File: Assignment33/Program2.java

import java.util.*;

//1.Write a program which accept N numbers from user and Display all such elements are divisible by 5

//Input N 6 
   
//         Elements : 85 66 3 80 93 88 
//Output: 85 80


class Program2
{   
    public static void Display(int Brr[])
    {
        int iCnt =0; 
        System.out.println("Divisble by 5 Numbesr are:");
        for(iCnt = 0; iCnt < Brr.length; iCnt++)
        {
          if(Brr[iCnt] % 5 == 0)
          {
             System.out.print(Brr[iCnt]+"\t"); 
          }
          System.out.println();
        }
    }
    public static void main(String Args[])
    {

      Scanner sobj = new Scanner(System.in);
      int iSize = 0; 
      int iCnt = 0;

      System.out.println("Enter the number of element:");
      iSize = sobj.nextInt();

      int Arr[] = new int[iSize]; //dynamic memory allocation for array
      
      System.out.println("Enter the Elements: ");
      for(iCnt =0; iCnt < iSize; iCnt++)
      {
        Arr[iCnt] = sobj.nextInt();
      }
      
      Display(Arr);
      sobj.close();

    }
}


